<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageBlur</class>
 <widget class="QMainWindow" name="ImageBlur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>1168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageBlur</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 61, 61);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Image Path</string>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPath">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 170, 255);
selection-color: rgb(0, 85, 255);
selection-background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,0">
      <item>
       <widget class="QLabel" name="setRange">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Blur Range</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="setTxtRange">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220);

</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 200, 200);
color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSub">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 200, 200);
color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,1">
      <item>
       <widget class="QLabel" name="setThread">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Thread count</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="setTxtThread">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBlur">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 170, 255);
selection-background-color: rgb(0, 170, 255);
color: rgb(255, 255, 255);
alternate-background-color: rgb(0, 170, 255);</string>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
